client/src/components/profile/components/SocialIcons.js

```diff
@@ -26,31 +26,51 @@ const propTypes = {
 
 function LinkedInIcon(linkedIn) {
   return (
-    <a href={linkedIn} rel='noopener noreferrer' target='_blank'>
+    <a
+      aria-label='Link to our LinkedIn'
```

Comment:
It's a link to a camper's LinkedIn page. I think it should be something like this:
 ```suggestion
       aria-label=`Link to ${username}'s LinkedIn`
```

`username` you can pass in `props` from a Camper component.

Reply:
I see, thank you! I thought they were FCC's social icons. I will implement the changes you requested when I have a bit of downtime.

Totally appreciate the time you're investing on this 🙇 !! Apologies for the back-and-forth.